docs/readme.md

```diff
@@ -1 +1,124 @@
-readme.file
+# Chicken & Fries
```

Comment:
Break the installation instructions out of this readme and into a readme that lives at the root of the repo. The readme in the docs folder starts where the table of contents begins.

Reply:
fixed it